properties-panel-extension/app/provider/magic/MagicPropertiesProvider.js

```diff
@@ -35,45 +20,53 @@ export default function MagicPropertiesProvider(propertiesPanel, translate) {
   // API ////////
 
   /**
-   * Return the tabs provided for the given element.
+   * Return the groups provided for the given element.
    *
    * @param {DiagramElement} element
    *
-   * @return {(Object[]) => (Object[])} tab entry middleware
+   * @return {(Object[]) => (Object[])} groups middleware
    */
-  this.getTabs = function(element) {
+  this.getGroups = function(element) {
 
     /**
      * We return a middleware that modifies
-     * the existing tabs.
+     * the existing groups.
      *
-     * @param {Object[]} entries
+     * @param {Object[]} groups
      *
-     * @return {Object[]} modified entries
+     * @return {Object[]} modified groups
      */
-    return function(entries) {
-
-      // add the "magic" tab
-      var magicTab = {
-        id: 'magic',
-        label: 'Magic',
-        groups: createMagicTabGroups(element, translate)
-      };
-
-      entries.push(magicTab);
-  
-      // show general + "magic" tab
-      return entries;
+    return function(groups) {
+
+      // Add the "magic" group
+      if(is(element, 'bpmn:StartEvent')) {
+        groups.push(createMagicGroup(element, translate));
+      }
+
+      return groups;
     }
   };
 
 
   // registration ////////
 
-  // register our custom magic properties provider
-  // use a lower priority to ensure it is loaded after
-  // the basic BPMN properties
+  // Register our custom magic properties provider.
+  // Use a lower priority to ensure it is loaded after
+  // the basic BPMN properties.
   propertiesPanel.registerProvider(LOW_PRIORITY, this);
 }
 
-MagicPropertiesProvider.$inject = [ 'propertiesPanel', 'translate' ]
\ No newline at end of file
+MagicPropertiesProvider.$inject = [ 'propertiesPanel', 'translate' ];
+
+// Create the custom magic group
+function createMagicGroup(element, translate) {
+
+  // create a group called "Magic properties".
+  var magicGroup = {
```

Comment:
We are using `const` in various places in this example, hence I propose to use it here as well